src/core/logical_layers/renderers/BivariateRenderer.tsx

```diff
@@ -190,7 +192,7 @@ export class BivariateRenderer extends LogicalLayerDefaultRenderer {
     }
   }
 
-  private _updateMap(
+  _updateMap(
```

Comment:
if it's not private we shall remove underscore?

Reply:
so I needed to have renderer similar to BivariateRenderer but a bit tweaked
that's why MCDARenderer extends BivariateRenderer
just not to repeat all the code 
and I override 2 methods I needed
can't override private method, so made it public

Reply:
ah, my bad, it should've been protected  🤦